README.md

```diff
@@ -1,70 +1,63 @@
+- JavaScript/React
+- React Router
+- React-redux
+- Redux-logger
+- CSS
+- Bootstrap
+- Webpack
+- Babel
+- Jest
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Setup
 
-### `yarn build`
+- Get the link of the repository: `https://github.com/kimothokamau/covid-metrics-capstone.git`
+- Clone it as `https://github.com/kimothokamau/covid-metrics-capstone.git` on a Terminal
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Usage
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Run `npm install` on a Terminal to install the modules
+- Run `npm start` on a Terminal to start the server 
+- Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Tests
 
-### `yarn eject`
+- Run `npm test` on a Terminal to run the test suite
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Authors
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+👨‍💻 **Kevin Kamau**
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+- GitHub: [@kimothokamau](https://github.com/kimothokamau)
+- LinkedIn: [Kevin Kamau](https://www.linkedin.com/in/kevinkamauk/)
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-## Learn More
+## 🤝 Contributing
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Thanks to [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for allowing me to use his theme.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Contributions, issues, and feature requests are welcome on the [issues page](https://github.com/kimothokamau/covid-metrics-capstone/issues).
 
-### Code Splitting
+## Show your support
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+Give a ⭐️ if you like this project!
 
-### Analyzing the Bundle Size
+## Acknowledgments
 
```

Comment:
- [ ] I would invite you to add the live demo link in the `readme` to view your project in less time.

Reply:
Done